public/src/main/java/com/revenuecat/purchases/utils/DateHelperDupe.kt

```diff
@@ -0,0 +1,33 @@
+package com.revenuecat.purchases.utils
+
+import java.util.Date
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.days
+
+/**
+ * TODO this is a dupe of DateHelper.kt in common module, but it's used in the public module so we can't remove it yet,
+ * nor make it public here because it would be accessible to everyone
+ */
+internal data class DateActiveDupe(val isActive: Boolean, val inGracePeriod: Boolean)
```

Comment:
We should remember we need to make these internal in the original classes as well once we've joined the other modules into one